data/net/minecraft/network/SkipPacketException.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/network/SkipPacketException net/minecraft/network/SkipPacketException
+	COMMENT Used to signify that a packet decoding error is not fatal. If a packet fails to decode, but {@link Packet#isSkippable} returns true, then this exception is thrown instead and {@link Connection} will log a message instead of closing the connection.
```

Comment:
> The encoder logger will log the error regardless.

Correct, but the logging is in regards to `Connection#exceptionCaught`. If it sees `SkipPacketException` it just logs a warning instead of terminating.

Reply:
Ah, I see. Still, in either case this would be an encoding error, not a decoding one.

Reply:
Yes, I've fixed that issue